pygmt/tests/test_clib_to_numpy.py

```diff
@@ -18,6 +18,18 @@
 
     _HAS_PYARROW = True
 except ImportError:
+
+    class pa:  # noqa: N801
+        """
+        A dummy class to mimic pyarrow.
+        """
+
+        @staticmethod
+        def timestamp(*args, **kwargs):
```

Comment:
Follow the signature of `pyarrow.timestamp`.
```suggestion
        def timestamp(unit: str, tz: str | None = None):
```